docs/io/unified_table_fits.rst

```diff
@@ -0,0 +1,625 @@
+
+- ``NaN`` for float columns.
+- Value of ``TNULLn`` for integer columns, as defined by the column
+  ``fill_value`` attribute.
+- Null string for string columns (not currently implemented).
+
+When the file is read back those elements are marked as masked in the returned
+table, but see `issue #4708 <https://github.com/astropy/astropy/issues/4708>`_
+for problems in all three cases. It is possible to deactivate the masking with
+``mask_invalid=False``.
+
+The FITS standard has a few limitations:
+
+- Not all data types are supported (e.g., logical / boolean).
+- Integer columns require picking one value as the NULL indicator. If
+  all possible values are represented in valid data (e.g., an unsigned
+  int columns with all 256 possible values in valid data), then there
+  is no way to represent missing data.
+- The masked data values are permanently lost, precluding the possibility
+  of later unmasking the values.
+
+``astropy`` provides a work-around for this limitation that users can choose to
+use. The key part is to use the ``serialize_method='data_mask'`` keyword
+argument when writing the table. This tells the FITS writer to split each masked
+column into two separate columns, one for the data and one for the mask.
+When it gets read back that process is reversed and the two columns are
+merged back into one masked column.
+
+.. doctest-skip::
+
+  >>> from astropy.table.table_helpers import simple_table
+  >>> t = simple_table(masked=True)
+  >>> t['d'] = [False, False, True]
+  >>> t['d'].mask = [True, False, False]
+  >>> t
+  <Table masked=True length=3>
+    a      b     c     d
+  int64 float64 str1  bool
+  ----- ------- ---- -----
+     --     1.0    c    --
+      2     2.0   -- False
+      3      --    e  True
+
+.. doctest-skip::
+
+  >>> t.write('data.fits', serialize_method='data_mask', overwrite=True)
+  >>> Table.read('data.fits')
+  <Table masked=True length=3>
+    a      b      c      d
+  int64 float64 bytes1  bool
+  ----- ------- ------ -----
+     --     1.0      c    --
+      2     2.0     -- False
+      3      --      e  True
+
+.. warning:: This option goes outside of the established FITS standard for
+   representing missing data, so users should be careful about choosing this
+   option, especially if other (non-``astropy``) users will be reading the
+   file(s). Behind the scenes, ``astropy`` is converting the masked columns
+   into two distinct data and mask columns, then writing metadata into
+   ``COMMENT`` cards to allow reconstruction of the original data.
+
+``astropy`` Native Objects (Mixin Columns)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to store not only standard `~astropy.table.Column` objects to a
+FITS table HDU, but also any ``astropy`` native objects
+(:ref:`mixin_columns`) within a `~astropy.table.Table` or
+`~astropy.table.QTable`. This includes `~astropy.time.Time`,
+`~astropy.units.Quantity`, `~astropy.coordinates.SkyCoord`, and many others.
```

Comment:
"many others"? Really? I can't think of "any other". Maybe just say "others that follow the protocol described in the next section"?